.github/workflows/build-test.yml

```
@@ -7,10 +7,25 @@ on:
     branches: main
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    outputs:
+      repo-settings: ${{ steps.filter.outputs.repo-settings }}
+    steps:
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            repo-settings:
+              - '**.md'
+              - '.github/**'
+              - '.husky/**'
+              - '.vscode/**'
```

그런데 이렇게 하면 README.md 파일과 함께 **.tsx 파일을 같이 변경되었을 때 실행이 안되지 않나요?!

---

```suggestion
            repo-settings:
              - '**/!(*.md)'
              - '!.github/**/*'
              - '!.husky/**/*'
              - '!.vscode/**/*'
```

이렇게 하고 밑에서

`if: ${{ needs.changes.outputs.repo-settings == 'true' }}` 로 바꾸면 될 것 같네요!

---

커교수의 날카로운 시선 & 그리고 조언까지 감사합니다! 수정했어요 🚀